book-library/script.js

```diff
@@ -77,9 +79,9 @@ function render() {
     cell4.appendChild(changeBut);
     let readStatus = "";
     if (myLibrary[i].check == false) {
-      readStatus = "Yes";
-    } else {
       readStatus = "No";
+    } else {
+      readStatus = "Yes";
```

Comment:
if statement was assigning yes when read checkbox was not checked and assisgning no when checked. Flipped assigning lines